src/univalue/test/object.cpp

```diff
@@ -239,33 +239,33 @@ BOOST_AUTO_TEST_CASE(univalue_object)
 
     strKey = "age";
     v.setInt(100);
-    BOOST_CHECK(obj.pushKV(strKey, v));
+    BOOST_CHECK(obj.pushKV(strKey, v);
 
```

Comment:
looks to me the right parenthesis has gone missing in all of these tests?

Reply:
you are right, the script I used got to aggressive :P

Reply:
fixed